pages/common/msfvenom.md

```diff
@@ -0,0 +1,24 @@
+# msfvenom
+
+> Manually generate payloads for metasploit.
+> More information: <https://github.com/rapid7/metasploit-framework/wiki/How-to-use-msfvenom>.
+
+- List Payloads:
+
+`msfvenom -l payloads`
+
+- List formats:
+
+`msfvenom -l formats`
+
+- Show payload options:
+
+`msfvenom -p {{payload}} --list-options`
+
+- Create an elf with a reverse tcp handler:
```

Comment:
You mention elf here, and exe in the next example. Both are not an apples-apples comparison. elf actually compares to pep for windows. I would recommend to say - Create an elf binary with a ...

Reply:
I know there are slight differences although this particular tool doesn't have the ability to export to a pep. I have made the change now